src/grocker/__main__.py

```diff
@@ -80,8 +80,11 @@ def purge(all_versions, including_final_images):
     '--push/--no-push', default=True,
     help='push the image',
 )
+@click.option(
+    '--manifest/--no-manifest', default=True, help='Hash provided from manifest',
+)
 @click.argument('release')
-def build(release, build_dependencies, build_image, push, **kwargs):
+def build(release, build_dependencies, build_image, push, manifest, **kwargs):
```

Comment:
I'd like to also be able to configure it through our shared configuration file.

Reply:
done, set to false by default